source/Paragraphs.h

```diff
@@ -0,0 +1,59 @@
+/* Paragraphs.h
+Copyright (c) 2024 by an anonymous author
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef PARAGRAPHS_H_
+#define PARAGRAPHS_H_
+
+#include "ConditionSet.h"
+
+#include <algorithm>
```

Comment:
What do we need `<algorithm>` for?
```suggestion
```

Reply:
That is one of the ways you can get the "pair" class. The "proper" way is to use utility, not algorithm. I've updated the code.

Reply:
![pair-header-utility](https://github.com/endless-sky/endless-sky/assets/116329264/4345c582-8cfb-4f20-b636-c064c5e64b16)